buildkite/bazelci.py

```diff
@@ -1447,6 +1447,9 @@ def PrepareRepoInCwd(print_cmd_groups, initial_setup=False):
                     eprint(ex)
 
             test_bep_file = os.path.join(tmpdir, _TEST_BEP_FILE)
+            # Create an empty test_bep_file so that the bazelci-agent can start to follow the file right away. Otherwise,
+            # there is a race between when bazelci-agent starts to read the file and when Bazel creates the file.
+            open(test_bep_file, 'w').close()
```

Comment:
Would this actually work? Doesn't bazel delete and create a new one, which would result in a different file handle?

Reply:
It works because the agent closes and opens the file in a loop.